src/Integration.Vsix.UnitTests/Analysis/IssueHandlerTests.cs

```diff
@@ -177,37 +168,38 @@ public void HandleNewIssues_HasFileLevelIssue_IssueIsReturned()
 
             // Assert
             notificationHandler.InvocationCount.Should().Be(1);
-            notificationHandler.UpdatedSnapshot.Issues.Count().Should().Be(1);
+            notificationHandler.UpdatedSnapshot.Issues.Count().Should().Be(0);
         }
 
         [TestMethod]
-        public void HandleNewIssues_NoIssues_NoIssuesReturned()
+        public void HandleNewIssues_SomeSuppressedIssues_IssuesGetMarkedCorrectly()
         {
             // Arrange
```

Comment:
There isnt really a good reason why I chose 4 issues instead of 2 but it felt better to me